packages/perseus/src/widgets/numeric-input/numeric-input.stories.tsx

```diff
@@ -1,99 +1,188 @@
+            },
+        },
+        labelText: {
+            control: {type: "text"},
+            description:
+                " Translatable Text; Text to describe this input. This will be shown to users using screenreaders.",
+            value: "What's the answer?",
+            table: {
+                type: {summary: "string"},
+            },
+        },
+        rightAlign: {
+            control: {type: "boolean"},
+            description: "Whether to right-align the text or not",
+            table: {
+                type: {summary: "boolean"},
+            },
+        },
         size: {
             options: ["normal", "small"],
             control: {type: "radio"},
             defaultValue: "normal",
+            description:
+                "Use size 'Normal' for all text boxes, unless there are multiple text boxes in one line and the answer area is too narrow to fit them.",
+            table: {
+                type: {summary: "string"},
+                defaultValue: {summary: "normal"},
+            },
+        },
+        static: {
+            control: {type: "boolean"},
+            description: "Always false.  Not used for this widget",
+            table: {
+                type: {summary: "boolean"},
+            },
+        },
+        // ApiOptions and linterContext are large objects and not particularly applicable to this story,
+        // so we're hiding them from view to simplify the UI.
+        apiOptions: {
+            table: {
+                disable: true,
+            },
+        },
+        linterContext: {
+            table: {
+                disable: true,
+            },
         },
     },
 };
 
-export const Question1 = (): React.ReactElement => {
-    return <RendererWithDebugUI question={question1} />;
-};
+export default meta;
 
-export const Interactive = (args: StoryArgs): React.ReactElement => {
-    const props = generateProps(args);
-
-    return <NumericInput {...props} />;
+const updateWidgetOptions = (
+    question: PerseusRenderer,
+    widgetId: string,
+    options: PerseusNumericInputWidgetOptions,
+): PerseusRenderer => {
+    const widget = question.widgets[widgetId];
+    return {
+        ...question,
+        widgets: {
+            [widgetId]: {
+                ...widget,
+                options: {
+                    ...widget.options,
+                    ...options,
+                },
+            },
+        },
+    };
 };
 
-export const Sizes = (args: StoryArgs): React.ReactElement => {
-    const smallProps = generateProps({...args, size: "small"});
-    const normalProps = generateProps({...args, size: "normal"});
-
-    return (
-        <div>
-            <label>
-                Small:
-                <NumericInput {...smallProps} />
-            </label>
-            <label>
-                Normal:
-                <NumericInput {...normalProps} />
-            </label>
-        </div>
-    );
+export const Default = (
```

Comment:
After setting up all the Storybook controls, I wasn't really sure if we needed any more stories as we can control most of these options on the fly now.

Reply:
Ah! Perhaps a story for each answerForm, as it IS a little frustrating to edit the json views for that. If possible, it'd be good to get an example of the pi answerForm with a note about how to trigger the "It seems like you've approximated pi" message.